src/client/Microsoft.Identity.Client/MsalException.cs

```diff
@@ -170,19 +172,82 @@ public override string ToString()
 
         // DEPRECATE / OBSOLETE - this functionality is not used and should be removed in a next major version
 
-        private const string ExceptionTypeKey = "type";
-        private const string ErrorCodeKey = "error_code";
-        private const string ErrorDescriptionKey = "error_description";
+        private class ExceptionSerializationKey
+        {
+            internal const string ExceptionTypeKey = "type";
```

Comment:
would it make the below if statement to be cleaner with a foreach of these constants were added to a dict? 

```cs
if (AdditionalExceptionData.TryGetValue(key, out string value))
{
     exceptionData[key] = value;
}
```

Reply:
The property names in object and in JSON are different. So simple loop won't suffice - has to use some dictionary mapping, so this seems simpler.